media/css/hubs/_common.scss

```diff
@@ -13,23 +13,23 @@ $color-link-blue-dark: #175a77;
 // Section titles
 
 .section-title {
-    @include font-size(24px);
+    @include font-size-level4;
     margin: 0 0 20px;
 
     span {
         @include box-decoration-break(clone);
         background-color: #000;
         color: #fff;
         line-height: 1.3;
-        padding: 0 5px 0 60px;
+        padding: 0 5px 0 40px;
     }
 }
 
 /* -------------------------------------------------------------------------- */
 // CTA link
 
 a.cta-link {
-    @include font-size(18px);
+    @include font-size(14px);
```

Comment:
The original ask for this copy was to be visually smaller than the body copy, but 16px in bold looks the same size as 18px regular. One downside of only ever using these mixins is it's now impossible to define a set font size that doesn't change depending on the viewport size, unless it's something that inherits the base size directly.

Still, I'm going to change this now as you suggest else I feel we'll just be spinning wheels on what's really a minor detail.